inspirehep/bat/pages/create_literature.py

```diff
@@ -151,6 +209,42 @@ def _thesis_info_population(input_data):
     )
 
 
+def _book_info_population(input_data):
+    WebDriverWait(Arsenic(), 10).until(
+        EC.visibility_of_element_located((By.ID, 'series_title'))
+    )
+    Arsenic().find_element_by_id('publisher_name').send_keys(
+        input_data['publisher-name']
+    )
+    Arsenic().find_element_by_id('publication_date').send_keys(
+        input_data['publication-date']
+    )
+    Arsenic().find_element_by_id('publication_place').send_keys(
+        input_data['publication-place']
+    )
+    Arsenic().find_element_by_id('series_title').send_keys(
+        input_data['book-title']
+    )
+    Arsenic().find_element_by_id('series_volume').send_keys(
+        input_data['book-volume']
+    )
+
+
+def _chapter_info_population(input_data):
+    WebDriverWait(Arsenic(), 10).until(
+        EC.visibility_of_element_located((By.ID, 'book_title'))
+    )
+    Arsenic().find_element_by_id('book_title').send_keys(
+        input_data['book-title']
+    )
+    Arsenic().find_element_by_id('start_page').send_keys(
+        input_data['page-start']
+    )
+    Arsenic().find_element_by_id('end_page').send_keys(
+        input_data['page-end']
+    )
```

Comment:
These two guys should use the imperative mood... but it won't be fixed in this PR.